studioGhibliApi.css

```diff
@@ -0,0 +1,71 @@
+body {
+    font-family: monospace;
+    background-color: lavender;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center
+}
+
+img {
+    width: 70%;
+    height: 150px;
+}
+
+#headingContainer {
+    display: flex;
+    flex-direction: row;
+    position: relative;
```

Comment:
Positioning `relative` with pixels will make your elements disappear on different screen sizes if you are not careful. This is what is making the header and other elements go out of view. Reexamine what we were asking for in terms of horizontal alignment here.